google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb

```diff
@@ -46,15 +46,15 @@ def retrieve_snapshot project, subscription, snapshot_name
   describe "Topic", :pubsub do
 
     it "should be listed" do
-      topics = pubsub.topics
+      topics = pubsub.topics max: 1000
```

Comment:
This will only return the first page, so you would probably want to call `.all` to get them all.